test/e2e/kubelet_stats.go

```diff
@@ -661,3 +695,36 @@ func (r *resourceMonitor) GetCPUSummary() nodesCPUSummary {
 	}
 	return result
 }
+
+func (r *resourceMonitor) GetMasterNodeCPUSummary(summaryPerNode nodesCPUSummary) nodesCPUSummary {
+	result := make(nodesCPUSummary)
+	var masterSummary containersCPUSummary
+	var nodesSummaries []containersCPUSummary
+	for node, summary := range summaryPerNode {
```

Comment:
Note that both `metrics_grabber_test` and `metrics_util` actually have full Node object there, so only kubelet stats needs changes.